webservices/docs.py

```diff
@@ -422,7 +422,7 @@
 
 SCHEDULE_B_BY_PURPOSE = '''
 Schedule B receipts aggregated by disbursement purpose category. To avoid double counting, memoed items are not included.
```

Comment:
This should say "Schedule B disbursements" or "Schedule B expenditures" or "Schedule B spending data" but they're not receipts.